hmpps-sqs-spring-boot-autoconfigure/src/main/kotlin/uk/gov/justice/hmpps/sqs/HmppsSqsConfiguration.kt

```diff
@@ -112,7 +113,12 @@ class HmppsSqsConfiguration {
   @Bean
   @ConditionalOnMissingBean
   @Conditional(ConditionalOnAuditQueueDefinition::class)
-  fun hmppsAuditService(hmppsQueueService: HmppsQueueService, objectMapper: ObjectMapper) = HmppsAuditService(hmppsQueueService, objectMapper)
+  fun hmppsAuditService(
+    hmppsQueueService: HmppsQueueService,
+    objectMapper: ObjectMapper,
+    @Value("\${spring.application.name:}") applicationName: String?,
+    @Value("\${audit.service.name:}") auditServiceName: String?,
```

Comment:
Is this a property defined elsewhere? We've tried to "namespace" all the properties for this lib under `hmpps.sqs` to try and make it easier to understand instead of various random property names. Not sure if this should also live under that namespace?

Reply:
I've now removed the additional override